packages/web/src/lib/api.ts

```diff
@@ -38,24 +38,24 @@ export async function getSuggestedParams(algocl: Algodv2): Promise<SuggestedPara
  * @param userParams a dict containing custom params defined by the user
  * @param s suggested transaction params
  */
-export async function mkTxParams(
-	algocl: Algodv2,
-	userParams: TxParams,
-	s?: SuggestedParams
+export async function mkTxParams (
+  algocl: Algodv2,
+  userParams: TxParams,
+  s?: SuggestedParams
 ): Promise<SuggestedParams> {
-	if (s === undefined) {
-		s = await getSuggestedParams(algocl);
-	}
+  if (s === undefined) {
+    s = await getSuggestedParams(algocl);
+  }
 
-	if (userParams.flatFee === undefined) {
-		s.flatFee = userParams.totalFee !== undefined;
-	}
-	s.fee = userParams.totalFee || userParams.feePerByte || ALGORAND_MIN_TX_FEE; // eslint-disable-line @typescript-eslint/prefer-nullish-coalescing
+  if (userParams.flatFee === undefined) {
+    s.flatFee = userParams.totalFee !== undefined;
+  }
+  s.fee = userParams.totalFee || userParams.feePerByte || ALGORAND_MIN_TX_FEE; // eslint-disable-line @typescript-eslint/prefer-nullish-coalescing
 
-	s.firstRound = userParams.firstValid || s.firstRound; // eslint-disable-line @typescript-eslint/prefer-nullish-coalescing
-	s.lastRound =
-		userParams.firstValid === undefined || userParams.validRounds === undefined // eslint-disable-line @typescript-eslint/prefer-nullish-coalescing
-			? s.lastRound
-			: Number(userParams.firstValid) + Number(userParams.validRounds);
-	return s;
+  s.firstRound = userParams.firstValid || s.firstRound; // eslint-disable-line @typescript-eslint/prefer-nullish-coalescing
+  s.lastRound =
+    userParams.firstValid === undefined || userParams.validRounds === undefined // eslint-disable-line @typescript-eslint/prefer-nullish-coalescing
+      ? s.lastRound
+      : Number(userParams.firstValid) + Number(userParams.validRounds);
+  return s;
```

Comment:
lot of formatting problems. Seams that you don't use `yarn format` or you are not using the changes in develop we pushed.

Reply:
I ran `yarn format` and updated the code.